### PR TITLE
Add links to code of conduct from Readme and Contributing

### PR DESCRIPTION
**Pull request by DeeDeeG**
# Context
- Follow-up to #490 
- Adds links to `CODE_OF_CONDUCT.md` from `README.md` and `CONTRIBUTING.md`

# Summary of Changes

- `CONTRIBUTING.md` now has a link to the code of conduct, and a paragraph explaining what the code of conduct is about.
- `README.md` now has a link to the code of conduct, and a sentence mentioning what the code of conduct is about.

# Checklist

- [ ] Tested Mobile Responsiveness
- [ ] Added Unit Tests
- [x] CI Passes
- [ ] Deploys to Heroku on test Correctly (Maintainers will handle)
- [ ] Added Documentation (Service and Code when required)

### CODE REVIEW

**Comment by DeeDeeG** [comment]
I'm not 100% sure about the wording, but it's complete, and I made some effort to make sure the wording was about right, given limited time. Feedback or changes are welcome, but I would also be comfortable merging this as-is if need be.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,3 +44,5 @@ Please cover any new code with specs. We prefer code to be covered using RSpec o
 
 ## Now What?
 Checkout our [Wiki](https://github.com/RefugeRestrooms/refugerestrooms/wiki) and specifically the [newcomers guide](https://github.com/RefugeRestrooms/refugerestrooms/wiki/Maintainers'-Manual-%5C--Newcomers'-Guide).
+
+Please also read our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md), which gives guidance on our standards of community and interaction. These are designed to keep things constructive, respectful and welcoming for all the people who contribute to our project. It also explains how/where to direct any complaints about unwelcome behavior.
```
Comment (DeeDeeG) [inline CONTRIBUTING.md]: "Please also read" could be changed to --> "See also"

File: README.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ For more information on how to contribute to Refuge Restrooms, or how the techno
 
 If you just want to get your environment set up for making changes locally and testing, you can head directly to [CONTRIBUTING.md](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CONTRIBUTING.md).
 
+See also our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md) for our standards of community, and who is responsible for enforcement, and where to direct complaints.
```
Comment (DeeDeeG) [inline README.md]: "enforcement" --> sounds too harsh? Or maybe it's just authoritative, in a way that is appropriate for framing a code of conduct.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mi-wood) [reply CONTRIBUTING.md]: I would leave it at: `Please also read our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md), which gives guidance on our standards of community and interaction.`. Anything else can be explained in the COC itself

File: README.md (hunk 2 above)
Comment (mi-wood) [reply README.md]: I might just c/p from above to here `Please also read our [Code of Conduct](https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/CODE_OF_CONDUCT.md), which gives guidance on our standards of community and interaction.`

**Comment by DeeDeeG** [comment]
Thanks for the feedback. Sounds good to me.

If there is a tad more discussion I'd feel like we could reach a real group consensus (always nice) and represent the feelings of the team as a whole, and/but in the interest of time I'm leaning toward implementing your suggestions verbatim.

If it gets to mid Sunday without updates I'll plan on doing that, since this should be merged "by the end of the week" (if I recall correctly?) -- for the Social Impact collection at GitHub. And of course we can update it as we go, and as I recall we have tended to improve README and CONTRIBUTING over time, so it's quite natural to continue doing so.

So I think we'll be fine on this, it's low pressure at this point. Just delaying 'til tomorrow to give room for further feedback and dialogue/suggestions. (If anyone at all who has contributed in the past or is thinking of doing so soon has comments/suggestions, I'd personally be open to hearing your feedback.)

**Pull request merged**